server/src/test/java/org/cloudfoundry/identity/uaa/provider/oauth/ExternalOAuthAuthenticationManagerTest.java

```diff
@@ -184,7 +186,7 @@ public void getExternalAuthenticationDetails_whenProviderIssuerMatchesUaaIssuer_
         throws ParseException, JOSEException {
         oidcConfig.setIssuer(tokenEndpointBuilder.getTokenEndpoint(IdentityZoneHolder.get()));
         expectedException.expect(InvalidTokenException.class);
-        expectedException.expectMessage("Could not verify token signature.");
+        expectedException.expectMessage("Invalid signature hash.");
```

Comment:
Is this error message something the code consumes internally, or is it returned to the user? If the former then it's fine, if the latter we then might need to look deeper to see if there's any unexpected effects (since the user interface has technically changed).